code/controllers/subsystems/ticker.dm

```diff
@@ -613,20 +613,12 @@ var/datum/controller/subsystem/ticker/SSticker
 			minds += player.mind
 
 /datum/controller/subsystem/ticker/proc/equip_characters()
-	var/captainless = TRUE
```

Comment:
This technically isn't a bugfix. 🤔

Reply:
it's annoying though, and served no functionality; one could argue that it is fixing something that bugs me, therefore it is a bug :^)